scss/civicrm/contact/pages/_export.scss

```diff
@@ -1,6 +1,9 @@
 .CRM_Export_Form_Select {
-  h2 {
-    padding-left: 20px;
+  padding-bottom: 5px;
+
+  .help {
```

Comment:
https://github.com/civicrm/org.civicrm.shoreditch/blob/develop/scss/civicrm/common/_help.scss
This file styles help section, do you need only margin change?